docs/contributing-to-rocr.rst

```diff
@@ -0,0 +1,93 @@
+.. meta::
```

Comment:
I don't recall seeing a contributing topic in the other refactored projects - why do we need this one?

Reply:
@lpaoletti It is there Leo. Refer to hipBLASlt, MIgraphX, hipTensor etc.